ios/PolyPodApp/Features/Feature.swift

```diff
@@ -28,7 +29,9 @@ class Feature {
         name = translations?.name ?? manifest.name ?? id
         author = translations?.author ?? manifest.author
         description = translations?.description ?? manifest.description
-        primaryColor = parseColor(hexValue: translations?.primaryColor ?? manifest.primaryColor)
+        let primaryColor = parseColor(hexValue: translations?.primaryColor ?? manifest.primaryColor)
+        self.primaryColor = primaryColor
```

Comment:
Why is primaryColor declared again?

Reply:
As a matter of fact, all uses of feature.primaryColor seem to have been substituted by feature.thumbnailColor. Is that possible?

Reply:
Xcode 12 didn't like me referencing another field in the constructor, so I had to introduce a temporary variable so I can use the same value twice. Bit stupid, in Xcode 13 it works as expected.

Reply:
As for `primaryColor` being substituded by `thumbnailColor`: That's only for the feature list card background. `primaryColor` is still being used for the nav bar colour.